### ISSUE TITLE
 1

### ISSUE THREAD

**Issue by a480615** [viesturz/NozzleAlign#5]
1

**Comment by viesturz** [comment]
Just to reply to the original content. 
Looks possible, the treads might interfere significantly however. 

A round head revet would work better: https://www.google.com/search?q=rivet%20round%20head
There are def advantages to this, making a test....

**Comment by a480615** [comment]
> 只是为了回复原始内容。看起来有可能，但胎面可能会受到显着干扰。
> 
> 圆头 revet 会更好： https://www.google.com/search?q=rivet%20round%20head 这样做有明显的优势，做一个测试......



> Just to reply to the original content. Looks possible, the treads might interfere significantly however.
> 
> A round head revet would work better: https://www.google.com/search?q=rivet%20round%20head There are def advantages to this, making a test....




Okay, thank you for your reply. Once I can successfully use NozzleAlign, I will proceed to test it. For this purpose, I've modeled a solution based on your design, involving a screw or rivet. The parts list roughly consists of two F695 bearings, one rivet, and one micro switch.
![8B9B36A0B8EEA2D98A101F150670A798](https://github.com/viesturz/NozzleAlign/assets/166965276/31a6e087-a369-49bf-9158-f898ed7c0477)
![9872CB2056E0260019C5CF14FF853465](https://github.com/viesturz/NozzleAlign/assets/166965276/d2c2768d-3476-4903-aeff-c1e91d56da9c)
![E9E07F0A2B42C039D83AE830B67C43CA](https://github.com/viesturz/NozzleAlign/assets/166965276/f736466d-bc38-4105-a052-1cc4111dc751)
https://github.com/viesturz/NozzleAlign/assets/166965276/ed8d150a-d50c-48ef-ae30-2b4d2bc69e07

**Comment by a480615** [comment]
> Just to reply to the original content. Looks possible, the treads might interfere significantly however.
> 
> A round head revet would work better: https://www.google.com/search?q=rivet%20round%20head There are def advantages to this, making a test....

I'm sorry, but currently I've only assembled the relevant hardware and cannot conduct actual usability tests. I believe you might have seen my response in another thread. Like others, I encountered an issue during the NozzleAlign center-finding process where the first X-axis movement triggering the limit switch resulted in an error.

My friends and I have discussed this issue extensively. Throughout this process, I tried adjusting parameters according to your troubleshooting suggestions, as well as switching the wiring of the micro switch between NC and NO, but none of these methods resolved the issue.

Finally, my friend informed me that when researching this issue, they saw that the boss of the Voron team seemed to have encountered the same issue as me (my machine's structure is similar to the hybrid CoreXY-Idex being developed by Voron). Perhaps it's a compatibility issue with this unique motion structure?

After tinkering with it all night, I've decided to temporarily abandon this approach and wait for good news from you or the Voron team. I won't give up on using this solution, but my expertise lies only in 3D modeling; I'm not proficient with coding, so I'll have to pause and wait.

It's also for the reasons mentioned above that I've closed the original thread. It's too reckless to give such casual suggestions before testing their usability.

Here's a video of my machine in action. It just needs your NozzleAlign to work perfectly:

https://github.com/viesturz/NozzleAlign/assets/166965276/06229119-1c1b-4c0c-a1a2-139ed9411811
